src/transitions/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { Fragment } from 'react'
+import Transition  from 'react-transition-group/Transition'
+import cc from 'classcat'
+import { transition } from '_utils/branding'
+import style from './style'
+
+export enum AnimationType {
```

Comment:
As it is, those constants are not available when importing the library in another project (because only the `default` export is available in the index file). I think they should be accessible when using the `Transition` component in another project.
You have two solutions:
- add the `AnimationType` in the main `src/index` file
- or attach them as a `static` to the `CustomTransition` class

I went with the second solution on my PR for Button and ItemChoice API, but it's your choice as my PR has not been merged yet.

Reply:
I attach TYPES as `static` and to make it work I was constraint to convert the stateless component to a class. 
In case of simple function, I tried to assign TYPES like this `CustomTransition.TYPES =  AnimationType ` but don't find a way to declare TYPES to typescript. If anyone has ideas to keep it stateless...